omod/pom.xml

```diff
@@ -44,6 +44,18 @@
 			<classifier>tests</classifier>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod-common</artifactId>
```

Comment:
If you wanted to increase the REST version, i would rather go for a non snapshot version. e.g 2.20.0
That way, we do not get into errors of trying to release the idgen module with snapshot dependencies.

Reply:
That being said, i like the fact that you were able to figure out how to resolve these errors when using the latest snapshot version of the rest module. 😊 
The other alternative would be for us to first release version 2.21.0 of the rest module.